hddm/models/hddm_stimcoding_nn.py

```diff
@@ -0,0 +1,193 @@
+            #print("Setting model to be non-informative")
+            kwargs['informative'] = False
+
+            # Add z if it is split parameter but not included in 'include'
+            if 'include' in kwargs and 'z' not in kwargs['include']:
+                kwargs['include'].append('z')
+            else:
+                #print('passing through here...')
+                if 'include' not in kwargs:
+                    kwargs['include'] = ['z']
+                else:
+                    pass
+
+            #print("Adding z to includes.")
+
+        # Get unique stimulus values for the stimcoding relevant column (has to be of length 2!)
+        self.stims = np.asarray(np.sort(np.unique(args[0][self.stim_col])))
+        assert len(self.stims) == 2, "%s must contain two stimulus types" % self.stim_col
+
+        super(HDDMnnStimCoding, self).__init__(*args, **kwargs)
+        #print(self.p_outlier)
+
+    def _create_stochastic_knodes(self, include):
+        knodes = super(HDDMnnStimCoding, self)._create_stochastic_knodes(include)
+        #print('knodes')
+        #print(knodes)
+        if self.drift_criterion:
+            knodes.update(self._create_family_normal_normal_hnormal('dc',
+                                                                     value = 0,
+                                                                     g_mu = 0,
+                                                                     g_tau = 3**-2,
+                                                                     std_std = 2))
+        return knodes
+
+    def _create_wfpt_parents_dict(self, knodes):
+        wfpt_parents = super(HDDMnnStimCoding, self)._create_wfpt_parents_dict(knodes)
+        
+        # SPECIFIC TO STIMCODING
+        if self.drift_criterion: 
+            wfpt_parents['dc'] = knodes['dc_bottom']
+
+        #print('wfpt parents: ')
+        #print(wfpt_parents)
+        return wfpt_parents
+
+    def _create_wfpt_knode(self, knodes):
+        
+        wfpt_parents = self._create_wfpt_parents_dict(knodes)
+        #print('wfpt parents in _create_wfpt_knode', wfpt_parents)
+        # Here we use a special Knode (see below) that either inverts v or z
+        # depending on what the correct stimulus was for that trial type.
+        
+        return KnodeWfptStimCoding(self.wfpt_nn, 
+                                   'wfpt', # TD: ADD wfpt class we need
+                                   observed = True, 
+                                   col_name = ['rt', 'response'], # Note: This is different from vanilla stimcoding class where it was set to col_name = 'rt',
+                                   depends = [self.stim_col],
+                                   split_param = self.split_param,
+                                   stims = self.stims,
+                                   stim_col = self.stim_col,
+                                   **wfpt_parents)
+class KnodeWfptStimCoding(Knode):
```

Comment:
Doesn't this already exist so that you can import it? Trying to avoid code duplication as much as possible.